test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -777,6 +808,10 @@ public int nodeAndClientId() {
             return nodeAndClientId;
         }
 
+        public boolean isMasterNode() {
```

Comment:
I'd prefer to use "isMasterEligible" here too. I mean, I'd prefer it everywhere but I think it'd be ok to start using it here.